apps/public-reference/pages/applications/preferences/select.tsx

```diff
@@ -0,0 +1,189 @@
+/*
+4.1 Preferences Introduction
+Instructions on how preferences work and their value
+*/
+import Link from "next/link"
+import Router from "next/router"
+import { Button, ErrorMessage, FormCard, ProgressNav, t } from "@bloom-housing/ui-components"
+import FormsLayout from "../../../layouts/forms"
+import { useForm } from "react-hook-form"
+import { AppSubmissionContext } from "../../../lib/AppSubmissionContext"
+import ApplicationConductor from "../../../lib/ApplicationConductor"
+import FormStep from "../../../src/forms/applications/FormStep"
+import { useContext, useMemo, useState } from "react"
```

Comment:
More pedantic changes, but "best practice" (and what we've been doing in other files) is to organize imports with third party imports first, then doing "local" imports to our packages.